src/Plugin_Command.php

```diff
@@ -820,15 +820,52 @@ public function get( $args, $assoc_args ) {
 	 * : If set, the plugin files will not be deleted. Only the uninstall procedure
 	 * will be run.
 	 *
+	 * [--all]
+	 * : If set, all plugins will be uninstalled.
+	 *
 	 * ## EXAMPLES
 	 *
 	 *     $ wp plugin uninstall hello
 	 *     Uninstalled and deleted 'hello' plugin.
 	 *     Success: Installed 1 of 1 plugins.
 	 */
 	public function uninstall( $args, $assoc_args = array() ) {
+
+		$all = Utils\get_flag_value( $assoc_args, 'all', false );
+
+		// Check if plugin names of --all is passed.
+		if ( ! ( $args = $this->check_optional_args_and_all( $args, $all ) ) ) {
+			return;
+		}
+
 		$successes = $errors = 0;
 		$plugins = $this->fetcher->get_many( $args );
+		if ( count( $plugins ) < count( $args ) ) {
+			$errors = count( $args ) - count( $plugins );
+		}
+
+		// If get all parameter from command, then uninstall all deactive plugins.
+		if ( $all ) {
```

Comment:
This block should be removed as it isn't needed after setting the args with `check_optional_args_and_all()`.